src/netgearpy/const.py

```diff
@@ -14,3 +14,7 @@
 
 LOGIN_BODY = """<M1:SOAPLogin xmlns:M1="urn:NETGEAR-ROUTER:service:DeviceConfig:1">
 <Username>{}</Username><Password>{}</Password></M1:SOAPLogin>"""
+
+GET_ATTACHED_DEVICES_BODY = (
+    """<M1:GetAttachDevice xmlns:M1="urn:NETGEAR-ROUTER:service:DeviceInfo:1" />"""
```

Comment:
I think this type of body is very standard.
Only the `GetAttachDevice` and `DeviceInfo:1` really changes.
Therefore you could consider making this a standard body (GET_BODY) and then filling in the service and method: https://github.com/MatMaul/pynetgear/blob/d9b34149b110cd91fbccc5fff7a285a1221af5dd/pynetgear/const.py#L153-L156